conf/tslint.yaml

```diff
@@ -10,7 +10,7 @@ rules: #
     object-literal-sort-keys: false
     arrow-parens: false
     no-console: false
-    prefer-const: true
+    prefer-const: false
```

Comment:
why don't we prefer constants?

Reply:
I mentioned this change in [my previous comment](https://github.com/FlowCrypt/flowcrypt-browser/pull/4026#issuecomment-932503485):

> also added one unrelated commit 4dfee3c which moves the prefer-const rule from tslint to eslint. The reason for that is that eslint would autofix that rule for us and tslint is deprecated.